components/automate-ui/src/app/components/resource-dropdown/resource-dropdown.component.scss

```diff
@@ -122,3 +122,7 @@ chef-dropdown {
   font-size: 12px;
   line-height: 1.6;
 }
+
+::ng-deep chef-checkbox .label-wrap {
+  overflow-wrap: anywhere;
```

Comment:
Safari still doesn't [fully support](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap#Browser_compatibility) `overflow-wrap`

![](https://user-images.githubusercontent.com/479121/91469403-598b6200-e861-11ea-9f8a-986b45fcbc7c.png)

So we'll probably need to settle for: `word-break: break-all`

![](https://user-images.githubusercontent.com/479121/91469535-80499880-e861-11ea-8c20-d21e507e66c0.png)

Reply:
I have fixed this.